tests/zfs-tests/tests/functional/write_dirs/setup.ksh

```diff
@@ -33,4 +33,6 @@
 
 verify_runnable "global"
 
-default_setup "$DISKS"
+DISK=${DISKS%% *}
```

Comment:
This seems entirely reasonable for testing with the CI where the DISKS are relatively small (4G).  But when testing using real multi-TB DISKS this could take unreasonably long.  What if we instead we used a fixed size file-vdev based pool.  It could be relatively small, say 2G, since these tests are primarily concerned with checking the ENOSPC handling.

Reply:
OK